doc/classes/CodeHighlighter.xml

```diff
@@ -18,7 +18,7 @@
 			<description>
 				Adds a color region such as comments or strings.
 				Both the start and end keys must be symbols. Only the start key has to be unique.
-				[param line_only] denotes if the region should continue until the end of the line or carry over on to the next line. If the end key is blank this is automatically set to [code]true[/code].
+				[param line_only] defines if the region should continue until the end of the line or carry over on to the next line. If the end key is blank this is automatically set to [code]true[/code].
```

Comment:
```suggestion
				If [param line_only] is [code]true[/code] or [param end_key] is an empty [String], the region does not carry over to the next line.
```
Assuming

Reply:
I'm going to transfer over what I described in CodeEdit here in full